src/usr/local/www/services_ntpd.php

```diff
@@ -60,8 +61,10 @@
 	unset($input_errors);
 	$pconfig = $_POST;
 
-	if (!is_numericint($_POST['ntpmaxpeers']) || ($_POST['ntpmaxpeers'] < 4) || ($_POST['ntpmaxpeers'] > $max_candidate_peers)) {
-		$input_errors[] = gettext("Max candidate pool peers must be a number between 4 and $max_candidate_peers");
+	if (!is_numericint($_POST['ntpmaxpeers']) ||
+	    ($_POST['ntpmaxpeers'] < $min_candidate_peers) ||
+	    ($_POST['ntpmaxpeers'] > $max_candidate_peers)) {
+		$input_errors[] = gettext("Max candidate pool peers must be a number between $min_candidate_peers and $max_candidate_peers");
```

Comment:
You shouldn't use variables inside `gettext()` strings. It should be like this instead:

    sprintf(gettext("Max candidate pool peers must be a number between %d and %d"), $min_candidate_peers, $max_candidate_peers);